### DISCUSSION TITLE
Who do I talk to to speed up my benefits?

### DISCUSSION THREAD

**Discussion by YifeiZhang123** [community/community#146878] [GitHub Education]
### Hi everyone,

I've passed the certification, but I didn't give me the relevant rights, I'm in a hurry to use it, who can help me...

**Comment by YifeiZhang123** [comment]
I just want to get my benefits,becasue i will use it today,plz,proTnT

**Comment by Chuofun** [comment]
It has been four days since my certification, but I still haven't received my student welfare authorization.

**Comment by davevad93** [comment]
Hi @YifeiZhang123 ,

GitHub no longer provide manual reviews, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @YifeiZhang123 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by YifeiZhang123** [reply]
Yes, I know this regulation, but why does it take so long Didn't I already turn in the materials and pass it

**Reply by YifeiZhang123** [reply]
If possible, can you help me apply for the benefits as soon as possible

**Reply by davevad93** [reply]
Hey @YifeiZhang123, I'm sorry but I can't help you with this because I'm not a GitHub employee. It takes some time because there are a lot of daily applications and it may take some time to review and process them. Don't worry, you'll get your benefits soon! 👍

**Reply by YifeiZhang123** [reply]
ok，i got it，ur so good bro

**Reply by davevad93** [reply]
If you want @YifeiZhang123 , you could raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and ask directly support.

**Comment by jonhubby** [comment]
Reach out to the certification team directly. They can likely help get your permissions sorted quickly.

**Reply by YifeiZhang123** [reply]
yeah,i want do this,but i dont know how to contact the team

**Reply by YifeiZhang123** [reply]
can you help me,guys
